model_servers/llamacpp_python/cuda/Containerfile

```diff
@@ -1,8 +1,7 @@
 FROM quay.io/opendatahub/workbench-images:cuda-ubi9-python-3.9-20231206
 WORKDIR /locallm
 COPY src .
-RUN pip install --upgrade pip
 ENV CMAKE_ARGS="-DLLAMA_CUBLAS=on"
```

Comment:
I suggest we also tackle https://github.com/containers/ai-lab-recipes/issues/243 here.